importer/package.json

```diff
@@ -7,10 +7,13 @@
     "jshint": "jshint --reporter=node_modules/jshint-stylish src || true",
     "clean": "find dist -type f ! -name '.keep' -delete && rm -rf dist/*",
     "generate:svg": "node generate.js --source=../assets --dest=./dist --extension=svg",
+    "generate:react": "node generate.js --source=../assets --dest=./dist --extension=svg --target=react",
     "generate:pdf": "node generate.js --source=../assets --dest=./dist --extension=pdf",
     "build:android": "npm run generate:svg && sh tools/vd-tool/bin/vd-tool -c -in ./dist -out ./dist && find ./dist/ -type f -name '*.svg' -delete && find ./dist/ -type f ! -name '*_selector.xml' -exec avocado {} + -exec replace '#212121' '@color/fluent_default_icon_tint' {} + && replace '\"http://schemas.android.com/apk/res/android\"' '\"http://schemas.android.com/apk/res/android\" android:autoMirrored=\"true\"' $(awk '$0=\"./dist/\"$0\".xml\"' rtl.txt)",
     "build:ios": "npm run generate:pdf",
+    "build:react": "npm run generate:react",
     "deploy:android": "npm run build:android && rm -rf ../android/library/src/main/res/drawable && mkdir ../android/library/src/main/res/drawable && find ./dist/ -type f -name \"*.xml\" -exec cp {} ../android/library/src/main/res/drawable \\; && npm run clean",
+    "deploy:react": "npm run build:react && rm -rf ../react/src/components && mkdir ../react/src/components && find ./dist/ -type f -name \"*.tsx\" -exec cp {} ../react/src/components \\; && npm run clean",
```

Comment:
* Please add a "build-react" job in pr.yml to run this command and another command that builds the react project for validation.
* Please add a "publish-react" job in publish.yml to run this command and another command that publishes the react library with version number in sync with the Android, iOS and Flutter libraries.
* Please include a `README.md` file in the `react` folder with documentation on the usage.

Reply:
These are all planned as part of a future update after we clean up some of the SVG files on the design side, but we are trying to make this available to test before the clean up can happen. I'll add issues to track the validation, publishing and documentation.